charts/registry/values.yaml

```diff
@@ -18,14 +18,16 @@
 # SPDX-License-Identifier: Apache-2.0
 ###############################################################
 
-
+nameOverride:
+fullnameOverride:
 # enables the default postgres database
 enablePostgres: true
 # enables the default keycloak identity provider
 # relies on a postgres instance
 enableKeycloak: false
 
 registry:
+  name: registry
```

Comment:
Whats the purpose of this? Couldn't find any usage of this.

Reply:
You are right. The property is not used. 
I removed this property.